server/core/basic_cluster.go

```diff
@@ -429,6 +444,9 @@ type StoreSetController interface {
 	PauseLeaderTransfer(id uint64) error
 	ResumeLeaderTransfer(id uint64)
 
+	SlowStoreEvicted(id uint64) error
```

Comment:
Can we reuse `PauseLeaderTransfer`/`ResumeLeaderTransfer`?

Reply:
That will make this scheduler conflicts with evict-leader-scheduler